app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -1286,9 +1288,27 @@ public void onBackPressed() {
             getSupportFragmentManager().popBackStack(NARROW,
                     FragmentManager.POP_BACK_STACK_INCLUSIVE);
         } else {
-            super.onBackPressed();
+            if(atHome){
```

Comment:
Suppose you are in private message and you presses back button, then it will display the message to press again to exit. But we don't want that. This function should work only when the user is on the main page. I hope you get use of it now. :)